readers/mutation_readers.cc

```diff
@@ -1119,12 +1119,11 @@ std::deque<mutation_fragment_v2> reverse_fragments(const schema& schema, reader_
 
 flat_mutation_reader_v2
 make_flat_mutation_reader_from_fragments(schema_ptr schema, reader_permit permit, std::deque<mutation_fragment_v2> fragments,
-        const dht::partition_range& pr, const query::partition_slice& query_slice) {
-    const auto reversed = query_slice.is_reversed();
+        const dht::partition_range& pr, const query::partition_slice& slice) {
```

Comment:
This patch changes the functions, but doesn't adjust any callers. Are there no callers which pass reverse slices here? Or does the adjustment happen in a different commit?